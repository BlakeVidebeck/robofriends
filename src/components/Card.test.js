import { shallow } from 'enzyme';
import Card from './Card';

it('Card renders correctly', () => {
	const mockRobot = {
		id: '1',
		name: 'JohnWick',
		email: 'johnwick@gmail.com',
	};
	const wrapper = shallow(<Card robot={mockRobot} />);
	expect(wrapper).toMatchSnapshot();
});
