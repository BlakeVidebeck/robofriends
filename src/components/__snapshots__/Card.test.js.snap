// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    height="200"
    src="https://robohash.org/1?size=200x200"
    width="200"
  />
  <div>
    <h2>
      JohnWick
    </h2>
    <p>
      johnwick@gmail.com
    </p>
  </div>
</div>
`;
